support-privacy.md

```diff
@@ -6,7 +6,7 @@ This policy does NOT apply to the Sandstorm software itself, which runs on your
 
 ## Introduction
 
-The Sandcats Support Service provides automatic updates for Sandstorm software that you run on your own server, "feature keys" which can be used to unlock features in that software, and email-based technical support.
+The Sandstorm Support Service provides automatic updates for Sandstorm software that you run on your own server, "feature keys" which can be used to unlock features in that software, and email-based technical support.
```

Comment:
To me, this is unclear -- I read it as if provides automatic updates for feature keys.

I'd prefer to see:

> The Sandstorm Support Service provides email-based technical support for Sandstorm, "feature keys" which can be used to unlock features within Sandstorm, and automatic updates for Sandstorm software that you run on your own server.

But I am OK as-is.